islands/IssuesList.tsx

```diff
@@ -0,0 +1,80 @@
+import { GitHubIssue } from '../lib/github.ts';
+import Card from '../components/Card.tsx';
+import { useState } from 'preact/hooks';
+
+type Props = {
+  repos: Array<string>;
+  issues: Array<GitHubIssue>;
+};
+
+type LabelProps = {
+  name: string;
+  color: string;
+}
+
+const IssuesList = ({repos, issues}: Props) => {
+  const [repoFilter, setRepoFilter] = useState(null);
+  const [labelFilter, setLabelFilter] = useState({});
+
+  const clearFilters = () => {
+    setRepoFilter(null);
+    setLabelFilter({});
+  }
+
+  const showLabel = (label: LabelProps) => {
+    return (
+      <span onClick={() => setLabelFilter(label)} style={"background-color: #".concat(label.color)} class="cursor-pointer font-bold mr-1 px-2 py-[0.125rem] rounded-xl text-white text-xxs whitespace-nowrap hover:text-black">
+        {label.name}
+      </span>
+    );
+  };
+
+  return (
+    <Card title="Current Issues">
+      <div class="flex justify-between my-4">
+        <div>
+          <select class="px-4 py-2" value={repoFilter} onChange={e => setRepoFilter(e.target.value)}>
```

Comment:
There was a type error here and I wasn't sure why, but found my way to [this MDN article](https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget) explaining that `currentTarget` is the element the `onChange` handler is being set on (the `select` in this case, which is what we want the `value` from), while `target` is the element that the event was triggered on (the `option`, which I guess also has a value, but either way).

The reason `target.value` was throwing a type error is because it's impossible to know exactly what type of element was clicked on to trigger the event, so we don't know whether it has a `value` property of not.

So yeah! `e.currentTarget.value` checks out here

```suggestion
          <select class="px-4 py-2" value={repoFilter} onChange={e => setRepoFilter(e.currentTarget.value)}>
```

Reply:
TIL 🤔 ✅